apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -59,7 +60,7 @@ export class UnconnectedTwoColumnActionBlock extends Component {
         <div style={styles.container}>
           {responsiveSize === 'lg' && (
             <div style={{float, width}}>
-              <img src={imageUrl} style={styles.image} alt={heading} />
+              <img src={imageUrl} style={styles.image} alt="" />
```

Comment:
This was reading the heading twice –– once above and once for the image, so I removed the image alt text